setup.py

```diff
@@ -25,7 +25,7 @@ def discover_test_suite():
           '': ['share/*', '*.rst', 'LICENSE'],
         },
         data_files=[
-          ('share/gist/', [
+          ('share/gist', [
```

Comment:
What error did this result in?

Reply:
I've to admit I didn't truly try without the patch...
When I first installed your current master branch (btw. any special reason the branches staging and bugfix aren't removed?) I did an ```python2 setup.py install``` and found the datafiles not installed, only after applying both patches I noticed, that I had to run a separate install_data, as such it might be completely cosmetic....

Reply:
Yeah, this change doesn't make any difference (at least on linux). So I don't want to merge this commit, but I am happy to merge the first commit in the PR.

Reply:
Then I'll create a new PR for just the first patch....